.github/workflows/python-package.yml

```diff
@@ -71,3 +72,71 @@ jobs:
             flags: unittests # optional
             fail_ci_if_error: true # optional (default = false)
             verbose: true # optional (default = false)
+
+
+  Integration:
```

Comment:
this might need some adjustment, I will try to push them this week After  my Eid vacation ends

Reply:
No rush, this won't go anywhere until after 5.1 anyway